common/postgresql/values.yaml

```diff
@@ -60,13 +60,12 @@ resources:
 backup:
   enabled: false
   repository: sapcc/backup-tools
-  image_version: v0.5.5
-  interval_full: "3 hours"
+  image_version: v0.5.6
+  interval_full: 1 hours
   os_auth_url: DEFINED-IN-REGION-SECRETS
-  os_auth_version: "3"
-  os_username: DEFINED-IN-REGION-SECRETS
-  os_user_domain: DEFINED-IN-REGION-SECRETS
-  os_project_name: DEFINED-IN-REGION-SECRETS
-  os_project_domain: DEFINED-IN-REGION-SECRETS
   os_region_name: DEFINED-IN-REGION-SECRETS
```

Comment:
not sure. the global part is fairly unclear so far. e.g arc does not declare the region in global. I would keep it in that fashion until we have a real global agreement